### PR TITLE
CLOUD-51308 itest used wrong cloudplatform

### PR DESCRIPTION
**Pull request by schfeca75**
@doktoric pls review

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/AzureTemplateCreationTest.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public void testAzureTemplateCreation(@Optional("it-azure-template") String azur
         TemplateRequest templateRequest = new TemplateRequest();
         templateRequest.setName(azureTemplateName);
         templateRequest.setDescription("AZURE template for integration testing");
```
Comment (doktoric) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/AzureTemplateCreationTest.java]: maybe the description should be also 'AZURE_RM template for integration testing'

Comment (schfeca75) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/AzureTemplateCreationTest.java]: Or maybe not :)

Comment (doktoric) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/AzureTemplateCreationTest.java]: or maybe yes.

**Comment by doktoric** [comment]
lgtm

**Pull request merged**